website/versioned_docs/version-1.0.0/input.md

```diff
@@ -85,6 +87,26 @@ styling for view containing the label, the input and the error message
 
 ---
 
+### `disabled`
+
+disables the input component
+
+|  Type   | Default |
+| :-----: | :-----: |
+| Boolean |  false  |
+
+---
+
+### `disabledInputStyles`
```

Comment:
```suggestion
### `disabledInputStyle`
```

Reply:
Totally missed this. Should be fixed now.